src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
What if you save information about fruits and quantity in map?

Reply:
Agree with the mentor, better to save fruits in Map<String, Integer> where the key is a name and the value is a quantity. Why? Cause getting fruit by name from the map is much faster